pkg/yamlexporter/exportutil.go

```diff
@@ -0,0 +1,85 @@
+package yamlexporter
+
+import (
+	"bufio"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/ghodss/yaml"
+	"github.com/pkg/errors"
+	"github.com/pravega/zookeeper-operator/pkg/apis/zookeeper/v1beta1"
+)
+
+// YAMLOutputDir where the zookeeper YAML resources will get generated
+var YAMLOutputDir string
+
+// CreateOutputSubDir creates a subdirectories where we want create the YAML file
+func CreateOutputSubDir(clusterName string, compName string) (string, error) {
+	fpath := filepath.Join(clusterName, compName)
+	return fpath, createDirIfNotExist(fpath)
+}
+
+// GenerateOutputYAMLFile writes YAML output for a resource
+func GenerateOutputYAMLFile(subdir string, depType string, data interface{}) error {
+
```

Comment:
Please remove the extraneous whitespace here. I think this all reads fine without.